questions/html-questions.md

```diff
@@ -11,5 +11,5 @@
 * What is progressive rendering?
 * Why you would use a `srcset` attribute in an image tag? Explain the process the browser uses when evaluating the content of this attribute.
 * Have you used different HTML templating languages before?
-* What's your process for addressing browser-specific rendering problems? Do you find that a certain browser is more challenging to work with than others?
-* Explain Why Table-less Layout Is Very Important?
+* What are ‘web workers’?
```

Comment:
sorry, how is it related to `html` ?

Reply:
Hi,
I mean:- https://www.w3schools.com/html/html5_webworkers.asp.

Reply:
Yes, it's html api, but it's not part of html as markup language. While here questions about markup / rendering and so on

Reply:
Webworkers is kind of api which could be used only with js - check the other questions to get the logic of grouping